core/templates/components/keyboard-shortcut-help/keyboard-shortcut-help-modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="modal-header">
+  <h3> Keyboard Shortcuts Help </h3>
+</div>
+
+<div class="modal-body">
+  <table>
+    <tr *ngFor="let item of KEYBOARD_SHORTCUTS | keyvalue">
```

Comment:
Prefer `const` here.

Reply:
It doesn't work, it raises an error when I change let to const

Reply:
<img width="403" alt="Screen Shot 2020-09-01 at 11 29 51 AM" src="https://user-images.githubusercontent.com/35385366/91891854-8f0bc300-ec46-11ea-9cc1-c417ecd7a890.png">

Reply:
Oh, strange. Guess AngularJS doesn't play nicely.